### DISCUSSION TITLE
Gohlke's wheels will be discontinued before July 2022

### DISCUSSION THREAD

**Discussion by martinfleis** [geopandas/geopandas#2745] [General]
Hey,

I just noticed that [Gohlke's wheels will be discontinued before July 2022](https://www.lfd.uci.edu/~gohlke/pythonlibs/). Once that happens, it will significantly affect installation of fiona on Windows for `pip` users. I think that this may even be seen as a bit of emergency situation on our side. 

One option would be to try to port wheel building infrastructure from pyogrio to fiona but I am not sure how simple that would be (@brendan-ward @jorisvandenbossche ?).

Another one would be to speed up #2225 and pyogrio 0.4 and maybe re-think the default engine selection and maybe even pip dependencies.

I am afraid that as soon as the wheels disappear, we will see a lot of installation troubles popping up.

**Comment by jorisvandenbossche** [comment]
I always had the idea that the work we did for pyogrio should be ported to fiona as well (we put a lot of effort in it, and we should ensure we benefit as much as possible from that). I also mentioned this already in Fiona (https://github.com/Toblerity/Fiona/discussions/1073#discussioncomment-2484730). I have been thinking to at least do a first step myself at some point (do a first basic port, and then let it up to the fiona devs to pick it up), but so this announcement might be a good reason to prioritize that a bit more. 

Regardless of that, it might indeed be a good reason to try to go faster with either a different default pip dependency or removing it as a pip default dependency.

**Comment by martinfleis** [comment]
> Regardless of that, it might indeed be a good reason to try to go faster with either a different default pip dependency or removing it as a pip default dependency.

I think we need at least one IO engine as a default dependency and depending on the state of Fiona wheels, we may want to have pyogrio there in 0.11.

**Comment by m-richards** [comment]
We now have the 0.11 release which mentions that pyogrio can be used instead of fiona, but we don't actually tell a windows user how they could actually use this information in practice (we still require fiona in the setup.py so `pip install geopandas` will try and build the fiona wheel from source).

A quick test, `pip install pandas shapely pyproj packaging pyogrio pygeos; pip install --no-deps geopandas` seems to work on windows, I'm not sure if there's a less hacky way (or if there needs to be, maybe fiona will have wheels before this becomes much of an issue)

**Comment by martinfleis** [comment]
We decided to wait for the change of the requirements until the Gohlke's wheels are completely gone and pyogrio wheels are a bit tested.

**Comment by martinfleis** [comment]
Update: the Gohlke's page now says "Archived" but seems to be still available. No idea for how long.
